src/ducks/actions.js

```diff
@@ -1,6 +1,8 @@
 import axios from 'axios';
 
 export const IS_AUTHED = 'IS_AUTHED';
+export const STUDENTS = 'STUDENTS';
+export const USER_INFO = 'USER_INFO';
```

Comment:
STUDENTS and USER_INFO could probably be more descriptive. Try to be more verbose on what precisely the action is doing. ex: UPDATE_REPO, ADD_STUDENT, DELETE_ACTIVITY, etc.